### ISSUE TITLE
Create international address label generator

### ISSUE THREAD

**Issue by riordan** [pelias/pelias#160]
Different countries have different ways of displaying placenames & addresses.

The issue raised by [pelias/api#351]

There has been some [discussion](https://github.com/whosonfirst/whosonfirst-data/issues/119) of using [Who's on First Names](https://github.com/whosonfirst/whosonfirst-names/blob/master/LABELS.md) to power this in the long-term.

But it seems like the easiest (and most collaborative) way to start would be by starting with [Open Cage Data's Address Formatting](https://github.com/OpenCageData/address-formatting) project.

**Comment by mtmail** [comment]
ping @freyfogle

**Comment by riordan** [comment]
:+1:

**Comment by freyfogle** [comment]
yes, would love you to join the effort on https://github.com/OpenCageData/address-formatting I have been slowly but surely plugging away there, usually one country a day. All contributions very welcome.
